xbmc/network/upnp/UPnPRenderer.cpp

```diff
@@ -700,9 +699,13 @@ CUPnPRenderer::OnSetVolume(PLT_ActionReference& action)
 {
     NPT_String volume;
     NPT_CHECK_SEVERE(action->GetArgumentValue("DesiredVolume", volume));
-    auto& components = CServiceBroker::GetAppComponents();
-    const auto appVolume = components.GetComponent<CApplicationVolumeHandling>();
-    appVolume->SetVolume((float)strtod((const char*)volume, NULL));
+    // From RenderingControl:3 Service Spec (2.2.16 Volume):
+    // The **unsigned integer** state variable represents the current volume
+    // setting of the associated audio channel. Its value ranges from a minimum of 0 to some device specific
```

Comment:
This log line is a little long, which might be hard to read by developers that require a large font to see. Can you wrap at 80 chars?

Reply:
done